build/gen-qt-projectfile.bat

```diff
@@ -66,42 +66,17 @@ echo       %1/share/locale/mscore_uz@Latn.ts \
 echo       %1/share/locale/mscore_vi.ts \
 echo       %1/share/locale/mscore_zh_CN.ts \
 echo       %1/share/locale/mscore_zh_TW.ts \
-echo       %1/share/locale/qt_ar.ts \
 echo       %1/share/locale/qt_bg.ts \

```

Comment:
Not needed as of Qt-5.9.1, but doesn't harm either, it'll get overwritten iv available in Qt